schedule/yast/maintenance/create_hdd_maintenance_migration.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
name of the file should be:
`create_hdd_maintenance_deregistered.yaml` to follow-up the name of the parent image.

Reply:
changed.